lib/services/message-matching.js

```diff
@@ -4,6 +4,12 @@ module.exports = {
   hasLink(message) {
     return linkRegex.test(message);
   },
+  hasTwitterLink(message) {
+    return linkRegex.test(message) && message.match(/twitter\.com/gi);
+  },
+  getLinks(message) {
+    return Array.from(message.matchAll(new RegExp(linkRegex, 'g')), (k) => k[0]);
```

Comment:
Instead of returning an array of strings, one could return an array of [URLs](https://nodejs.org/api/url.html), this would allow you to query the hostname for example, so instead of using a regex in `hasTwitterLink` you could essentially do `someurlObject.hostname === 'twitter.com'`